In Python 2.5, the with statement needs to be imported from
__future__.

diff a/test/split-testfile.py b/test/split-testfile.py
--- a/test/split-testfile.py
+++ b/test/split-testfile.py
@@ -5,6 +5,7 @@
 # Jansson is free software; you can redistribute it and/or modify
 # it under the terms of the MIT license. See LICENSE for details.
 
+from __future__ import with_statement
 import os
 import sys
 from optparse import OptionParser
